audio/filter/af_lavrresample.c

```diff
@@ -149,6 +150,11 @@ static int af_open(struct af_instance *af)
 
     s->converter = mp_aconverter_create(af->global, af->log, &s->opts);
 
+    if (s->deprecation_warning) {
```

Comment:
sorry, I didn't catch this earlier, but I incorrectly see this warning with 

    mpv  /home/kevmitch/mm/tmp/ZT.mkv  --config-dir=config --speed=1.1 --audio-pitch-correction=no

where `config/input.conf` contains 

    a cycle_values audio-device null auto

hit `a`